common.blocks/uri/uri.js

```diff
@@ -85,7 +85,7 @@ Uri.prototype.parseUri = function(str) {
     DO NOT split parser regex into parts because it can seriously affect performance!
     jsHint maxlen changed to fix maxlength warning at this line.
```

Comment:
it'd be great to provide exception for jshint locally just for this line, which will be also self-descriptive (so no need for comment afterwards)